index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="styles.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+	<div class="about-container">
+    <header class="menu-bar">
+      <img src="img/My-Logo.svg" alt="logo">
+      <img src="img/Union.svg" alt="Hamburger">
+    </header>
+    <article class="about">
+      <div class="intro">
+        <h2>I’m Victor Ozoede Glad to see you!</h2>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through   some of my work and experience! If you like what you see and have a project you need coded, don’t   hestiate to contact me.</p>
+      </div>
+      <a class="connect" href="#">let's connect</a>
+      <div class="social-icons">
+        <a href="#"><img src="./img/Vector.svg" alt="twitter"></a>
+        <a href="#www.linkedin.com/in/chukwuebuka-ozoede-46616a219"><img src="./img/Linkedin-icon.svg" alt="linkedin"></a>
+        <a href="#"><img src="./img/Vector1.svg" alt="mail"></a>
+        <a href="#https://github.com/VictorChukwuebuka"><img src="./img/Vector2.svg" alt="github"></a>
+        <a href="#"><img src="./img/Vector3.svg" alt="two fingers"></a>
+      </div>  
```

Comment:
- [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).
- [ ] Please wrap it with the section `The container of the section is a section tag`, kindly move our article inside the section tag.